index.js

```diff
@@ -53,7 +53,12 @@ module.exports = function(filepath) {
     });
 
     define(obj, 'dir', function() {
-      return path.dirname(filepath);
```

Comment:
shouldn't we try to do the same thing as node here? Or is this similar to how node handles it?

Reply:
This logic matches the path.parse tests in this repo.  I suppose I can dig into their code to see the logic.

Reply:
They don't actually use other `path.*` methods within their impl and instead parse everything manually but this is the behavior exhibited so I think we are good.  The updates to the tests exposed this problem.